staging/src/k8s.io/sample-controller/controller.go

```diff
@@ -103,6 +105,10 @@ func NewController(
 	eventBroadcaster.StartStructuredLogging(0)
 	eventBroadcaster.StartRecordingToSink(&typedcorev1.EventSinkImpl{Interface: kubeclientset.CoreV1().Events("")})
 	recorder := eventBroadcaster.NewRecorder(scheme.Scheme, corev1.EventSource{Component: controllerAgentName})
+	ratelimiter := workqueue.NewMaxOfRateLimiter(
+		workqueue.NewItemExponentialFailureRateLimiter(5*time.Millisecond, 1000*time.Second),
+		&workqueue.BucketRateLimiter{Limiter: rate.NewLimiter(rate.Limit(50), 300)},
```

Comment:
bit of a nite, but the limit here has changed: https://github.com/kubernetes/client-go/blob/435ce6f2a8ef03eb6920e66f889b13a1755f89df/util/workqueue/default_rate_limiters.go#L43

the default previous was:

```
	return NewMaxOfRateLimiter(
		NewItemExponentialFailureRateLimiter(5*time.Millisecond, 1000*time.Second),
		// 10 qps, 100 bucket size.  This is only for retry speed and its only the overall factor (not per item)
		&BucketRateLimiter{Limiter: rate.NewLimiter(rate.Limit(10), 100)},
	)
```

It looks like `DefaultControllerRateLimiter` is *not* deprecated - can we continue to use this, but with `NewRateLimitingQueue` rather than `NewNamedRateLimitingQueue`?

Reply:
Sorry yes we bump QPS and Burst to match other places we updated things ( [[1]](https://github.com/kubernetes/kubernetes/pull/105520/files#diff-7e82ada63a2e6664b803ac3ccdb84759ac97338ce807df63be2f81e3c9e0ba9cR297) [[2]](https://github.com/tektoncd/cli/pull/1693/files#diff-77ab2bc2d94c7936f23583ec9247458d4fdecce2c03b319fc88a67ef128e2d03R184-R185) [[3]](https://github.com/argoproj/argo-workflows/pull/11603/files#diff-cae57e8992eaf3f7bc489227fb7743bf2ff3c282948735021e134cfecbbdc111R380-R381) )